src/parse.rs

```diff
@@ -1084,6 +1080,30 @@ impl<'a> RawParser<'a> {
         None
     }
 
+    fn char_colon(&mut self) -> Option<Event<'a>> {
+        let beg = self.off;
+        let limit = self.limit();
+        let data = &self.text[..limit];
+        // if end is less than three chars away, can't be a URI
+        if limit <= beg + 3 {
+            return None;
+        }
+        if &data[beg+1..beg+3] != "//" {
+            return None;
+        }
+        let scheme_len = backtrack_uri_scheme(&data[..beg]);
+        if scheme_len != None {
+            let loc_len = scan_uri_no_scheme(&data[beg+3..]);
+            if loc_len == 0 { return None }
+            let uri = &data[beg-scheme_len.unwrap()..beg+3+loc_len];
+            self.off -= scheme_len.unwrap();
+            self.state = State::Literal;
+            Some(self.start(Tag::AutoLink(Borrowed(uri), scheme_len.unwrap() as u8), beg+uri.len()-scheme_len.unwrap(), 0))
```

Comment:
Umm so what happens here are you sending it back to the beginning of the line after running this parse?